projects/components/src/textarea/textarea.component.ts

```diff
@@ -29,6 +30,9 @@ export class TextareaComponent implements OnInit {
   @Input()
   public disabled: boolean | undefined;
 
+  @Input()
+  public rowsCount: number = 0;
```

Comment:
nit: `rowCount` (or `rows`)

Reply:
done. I need to replace the default count because 0 was failing.